pyproject.toml

```diff
@@ -115,7 +115,8 @@ filterwarnings = [
   'ignore:.*You are using pyarrow version',
   'ignore:.*but when imported by',
   'ignore:Distributing .*This may take some time',
-  'ignore:.*The default coalesce behavior'
+  'ignore:.*The default coalesce behavior',
+  'ignore::RuntimeWarning'
```

Comment:
this seems to broad to turn off globally, can we just catch it in the code?

Reply:
fair enough 🙈